### DISCUSSION TITLE
Improve scrolling experience in flyout when multiple devices exist

### DISCUSSION THREAD

**Discussion by beppe9000** [File-New-Project/EarTrumpet#699] [Ideas]
When scrolling the sidebar there should be some check in place to prevent accidental change of the volume of an item if the scroll overflows.

In practical terms the scroll event on the sliders needs to be canceled if the mouse pointer did not move since the last vertical scroll event on the container.

![example gif](https://i.imgur.com/QXLn9X9.gif)

**Comment by beppe9000** [comment]
volume should only change if the scroller is focused. idk if still like that

**Comment by HaArD2000** [comment]
Still does this in 2.1.8.0 I accidentally lower the volume when trying to scroll down in the list all the time.

If your stay on the left side where the icons are it will not affect the volume levels.
